internal/cmd/pipeline/command_update.go

```diff
@@ -44,6 +54,7 @@ func (c *command) newUpdateCommand(enableSourceCode bool) *cobra.Command {
 			"If <secret-value> is empty, the named secret will be removed from Stream Designer.")
 	}
 	cmd.Flags().Bool("activation-privilege", true, "Grant or revoke the privilege to activate this pipeline.")
+	cmd.Flags().Bool("update-schema-registry", false, "Update the pipeline with the latest schema registry cluster.")
```

Comment:
since it's a bool flag, it makes it feel like we can do `--update-schema-registry=false` to `unset` the schema-registry, but actually it won't.

In general, since both ksql and sr are not optional at creation and can be updated later, I feel we should also consider the story of `unset`
* if we can unset ksql/sr, then this update command probably need new flag to indicate `--remove-ksql-cluster` and `--remove-schema-registry`
* if we do not want to allow unset after it's set once, then we may have some communication work to do (since feels a bit odd why such restriction exists)

Reply:
YEs, i had a chat about that with Randall on `unset`. I don't think our APIs can support that at this point of time. Maybe we can handle that separately?

Reply:
When the API supports it, we can check to see if the user has modified the `--update-schema-registry` flag, and set/unset depending on if the value is true/false.

Reply:
We may want to consider renaming to `--set-schema-registry`?

Reply:
before API supports it, then `--update-schema-registry=false` or `--set-schema-registry=false` will be no-op? How should we communicate that to make it clearer?

Reply:
I don't expect any customers to be passing `--set-schema-registry false` before this change is made, so I'd be ok with making the change I suggested above as a breaking change. (No need to communicate anything, but let's wait until the API change is made before making this change in the CLI?)

Reply:
Sorry, maybe my comment was confusing. We dont have a mechanism to unset any of the configs. Ex: once the user sets a ksql cluster for a pipeline they can only change it to a new ksql cluster but cant remove it. This is the behavior for all of the metadata configs. so when when user calls with  `--update-schema-registry=false`, we just don't update the pipeline with new/updated sr cluster.

Reply:
Will we ever support it someday? It feels odd to me why we won't, so I assume someday we will want to either update/refresh ksql and SR, or disable/remove ksql and sr.

So we are just deciding what to start with for now that can be easily adapted later. If we can ignore users passing us `false` for now (which is no op today) and later support it (which will unset the SR). Then it feels more accurate to use `--enable-schema-registry` than `--update-schema-registry` or `--set-schema-registry`? The later doesn't feel like a bool flag. (With SR, users don't provide the `lsrc-id` so `update` or `set` doesn't feel very natual)

Reply:
As discussed offline, We will add a new flag to unset the sr cluster when our backend API supports that and we can continue with the current approach to update the cluster with `--update-schema-registry` flag.